easy_phi/auth.py

```diff
@@ -262,24 +267,143 @@ def get(self):
 
     post = get
 
+_user_cache = None
+
+
+class PasswordAuthAPIHandler(tornado.web.RequestHandler):
+    user = None
+    password = None
+
+    # this method is static to be used by check_passwords, which is in turn
+    # static to be used by PasswordAuthLoginHandler without class instantiation
+    @staticmethod
```

Comment:
What happens in the file is not there? Can we create that file manually on the first launch and display a warning that it's empty?